OrbitGl/GpuTrack.cpp

```diff
@@ -238,6 +241,10 @@ void GpuTrack::OnTimer(const Timer& timer) {
   if (timer.m_End > max_time_) max_time_ = timer.m_End;
 }
 
+std::string GpuTrack::GetTooltip() const {
+  return "Shows scheduling and execution times for selected GPU calls.";
```

Comment:
Same here, suggest to get rid of "."

I would change "calls" to "job submissions".